### PR TITLE
Mood tracker for twilight alley + Calming cliffs activity

### PR DESCRIPTION
**Pull request by vineam03**
"rigorously tested"

- Mood points are calculated at the end of twilight alley after a series of questions in which users can respond with emoji.

- Each emoji has a point value assigned, green = 3, yellow = 2, red = 1.
 
- If the total point value is under 13 OR if any red emojis were selected at anytime during the session a "flag" gets set which will be used later on

- Calming cliffs is still work in progress, maybe integrate with audio. I feel like it may not be enough as a standalone activity 

Let me know if you have any questions comments or concerns. 

_"It works on my machine" - Anonymous_

Very respectfully,

Vin

### CODE REVIEW

File: lib/TranquilForestLandingPage.dart [hunk 1]
```diff
@@ -62,16 +62,24 @@ class TranquilForestLandingPage extends StatelessWidget {
...
+                  'Journal some of your thoughts and track your mood.',
                   builder: (context) => TwilightAlleyIntro(),
                   // colors
                   cardColor: Colors.white.withOpacity(0.9), shadowColor: Colors.black.withOpacity(0.3), iconBackgroundColor: Colors.teal[700],iconColor:  Colors.white, textColor: Colors.teal[700], subTextColor: Colors.teal[600]
               ),
 
+              // Activity for calming visualization
+              activityCard('Calming Cliff', context, Icons.landscape,
+                  'Calm yourself and realize that there is so much out there.',
+                  builder: (context) => CalmingCliffsIntro(),
```
Comment (Kenneth-W-Chen) [inline lib/TranquilForestLandingPage.dart:75]: You're missing the import for this function

File: lib/screens/activities/calmingcliffintro.dart [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                ),
+                foregroundColor: Colors.deepOrangeAccent,
+              ),
+              onPressed: () {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(
+                      builder: (context) => const CalmingCliffsActivity()),
+                ).then((value) {
+                  setActivityCompleted(Future.value(value));
```
Comment (Kenneth-W-Chen) [inline lib/screens/activities/calmingcliffintro.dart:171]: This is receiving null. Not sure if you're still working on that, but you might be forgetting to pass a value with `Navigator.pop()` in the `CalmingCliffsActivity()`.

**Review by Kenneth-W-Chen — CHANGES_REQUESTED** [review]
I think it's fine as a standalone activity. Most of it looks like it works except for the import and `Navigator.pop()` not returning a value properly.

File: lib/TranquilForestLandingPage.dart (hunk 1 above)
Comment (vineam03) [reply lib/TranquilForestLandingPage.dart:75]: https://github.com/Kenneth-W-Chen/mindfulness-app/pull/26/commits/a5cd68340579aa0d82febee6ef93fababcedda50

File: lib/screens/activities/calmingcliffintro.dart (hunk 2 above)
Comment (vineam03) [reply lib/screens/activities/calmingcliffintro.dart:171]: https://github.com/Kenneth-W-Chen/mindfulness-app/pull/26/commits/99a3c26d1480c9d84845c830e6a27e2cce627bbb

**Review by Kenneth-W-Chen — APPROVED** [review]
lgtm

**Pull request merged**